README.md

```diff
@@ -35,6 +38,10 @@ For other systems, please see documentation:
 
 _Note: You will need to use [Node](https://nodejs.org/en/) v12 or higher._
 
+#### nvm
+
+Building PatternFly Elements requires specific versions of Node.js.  To automate usage of the correct version, it is recommended that contributors install [nvm](https://github.com/nvm-sh/nvm#readme).  With nvm installed, your terminal will automatically switch to the correct Node.js version when you `cd` into the PatternFly Elements repository.
```

Comment:
Does nvm switch versions automatically out of the box?  I thought it was a plugin that did that.  I could be wrong though.

Reply:
Yeah, just looked into it and you're right.  [avn](https://github.com/wbyoung/avn) can handle it, or you can use [shell-specific nvm integration](https://github.com/nvm-sh/nvm#deeper-shell-integration) (ex: [bash](https://github.com/nvm-sh/nvm#bash).

Reply:
```suggestion
Building PatternFly Elements requires specific versions of Node.js.  To automate usage of the correct version, it is recommended that contributors install [nvm](https://github.com/nvm-sh/nvm#readme) as well an automatic version switcher like [avn](https://github.com/wbyoung/avn) or [shell-specific nvm integration](https://github.com/nvm-sh/nvm#deeper-shell-integration).  With those installed, your terminal will automatically switch to the correct Node.js version when you `cd` into the PatternFly Elements repository.
```